[docs] #193: Update instructions page, add a page on modes (#200)
hyperledger/iroha-2-docs: src/guide/configure/modes.md

# Public and Private Blockchains

> Iroha can be compiled in two modes: **public** and **private**. The mode
> determines the set permissions granted to users as well as how peers can
> join the network.

Reviewer: ```suggestion
Iroha can be ran in a variety of configurations. Different permission sets allow you as the administrator of your own network to decide what criteria must be met in order for some transaction to be accepted. For ease of use, we provide two major sets of permissions: called a _private_ and _public_ permission sets. These need to be added into the `genesis.json` before you start an Iroha peer.  
```

> Below we will outline the differences between these two modes.

Reviewer: ```suggestion
Below we outline the major differences in these two use cases.
```
